packages/codemod/src/test/__testfixtures__/remove-anthropic-facade.input.ts

```diff
@@ -0,0 +1,14 @@
+// @ts-nocheck
+import { Anthropic } from '@ai-sdk/anthropic';
+
+const anthropic = new Anthropic({
+  apiKey: 'key',
+  baseURL: 'url',
+  headers: { 'custom': 'header' }
+});
+
+const model = anthropic.chat('claude-3', {
```

Comment:
Here (and for other providers) is another example of the same issue mentioned in https://github.com/vercel/ai/pull/3641#discussion_r1838830113

While we can easily (as the codemod currently does) look for method calls on the provider instance and correct them to create a model with the provider instance we track from above, this is just the tip of the iceberg in terms of potential usages needing correction.

Whatever we think re: the other PR we should likely apply to the approach we use here. Currently the codemod does the trivial easy correction.

Reply:
we should be careful about the method name changes, some are needed. the provider constructor might be enough

Reply:
Updated to only edit the ctor.